Dockerfile

```diff
@@ -2,6 +2,8 @@
 FROM docker.io/lambci/lambda:build-provided.al2
 
 ARG RUST_VERSION=1.54.0
+RUN yum -y update
+RUN yum -y remove kernel-devel-4.14.203-156.332.amzn2
```

Comment:
Can you explain please, why do we need to remove `kernel-devel` package?

Reply:
The yum update installs the latest kernel-devel package but does not remove the previous version. 
The previous version is not required and contains security vulnerabilities. 
We need to remove it explicitly.